### PR TITLE
Metadata test pass

### PR DESCRIPTION
**Pull request by rasapala**
Adding get model metadata for mediapipe graph type.

### CODE REVIEW

File: src/kfs_frontend/kfs_grpc_inference_service.cpp [hunk 1]
```diff
@@ -410,6 +420,36 @@ Status KFSInferenceServiceImpl::buildResponse(
     return StatusCode::OK;
 }
 
+#if (MEDIAPIPE_DISABLE == 0)
+Status KFSInferenceServiceImpl::buildResponse(
+    MediapipeGraphDefinition& mediapipeGraphDefinition,
+    KFSModelMetadataResponse* response) {
+
```
Comment (atobiszei) [inline src/kfs_frontend/kfs_grpc_inference_service.cpp:427]: Unnecessary empty lines

File: src/test/get_mediapipe_graph_metadata_response_test.cpp [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+    EXPECT_EQ(response.versions().at(0), "1");
+
+    EXPECT_EQ(response.platform(), "OpenVINO");
+
+    EXPECT_EQ(response.inputs_size(), 2);
+    auto firstInput = response.inputs().at(0);
+    EXPECT_EQ(firstInput.name(), "in");
+    EXPECT_EQ(firstInput.datatype(), "INVALID");
+    auto secondInput = response.inputs().at(1);
+    EXPECT_EQ(secondInput.name(), "in2");
```
Comment (atobiszei) [inline src/test/get_mediapipe_graph_metadata_response_test.cpp:90]: Add checks for empty shape

**Review by atobiszei — APPROVED** [review]

File: src/test/get_mediapipe_graph_metadata_response_test.cpp [hunk 3]
```diff
@@ -0,0 +1,104 @@
+//*****************************************************************************
+// Copyright 2020 Intel Corporation
```
Comment (atobiszei) [inline src/test/get_mediapipe_graph_metadata_response_test.cpp:2]: Coyright year

**Review by michalkulakowski — APPROVED** [review]

File: src/kfs_frontend/kfs_grpc_inference_service.cpp (hunk 1 above)
Comment (rasapala) [reply src/kfs_frontend/kfs_grpc_inference_service.cpp:427]: Ok.

File: src/test/get_mediapipe_graph_metadata_response_test.cpp (hunk 2 above)
Comment (rasapala) [reply src/test/get_mediapipe_graph_metadata_response_test.cpp:90]: Ok.

**Pull request merged**